app/javascript/src/QuickStarts/components/CustomCatalog.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+import { useContext, useMemo } from 'react'
+import {
+  QuickStartCatalogHeader,
+  QuickStartCatalogSection,
+  QuickStartContext,
+  QuickStartTile,
+  getQuickStartStatus,
+  LoadingBox
+} from '@patternfly/quickstarts'
+import {
+  Gallery,
+  GalleryItem,
+  Text,
+  TextContent
+} from '@patternfly/react-core'
+
+import type {
+  QuickStart,
+  QuickStartContextValues } from '@patternfly/quickstarts'
+
+import './QuickStartContainer.scss'
+
+export const CustomCatalog: React.FC = () => {
+  const { activeQuickStartID, allQuickStartStates, allQuickStarts, loading } =
+    useContext<QuickStartContextValues>(QuickStartContext)
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention -- Following PF team guidelines
+  const CatalogWithSections = useMemo(
+    () => (
+      <>
+        <QuickStartCatalogSection>
+          <TextContent>
+            <Text className="catalog" component="h2">3scale API Management features</Text>
+          </TextContent>
+          <Gallery hasGutter className="pfext-quick-start-catalog__gallery">
+            {allQuickStarts!
+              .filter((quickStart: QuickStart) => quickStart.metadata.threescaleAPIManagementFeatures)
+              .map((quickStart: QuickStart) => {
+                const {
+                  metadata: { name: id }
+                } = quickStart
+                return (
+                  <GalleryItem key={id} className="pfext-quick-start-catalog__gallery-item">
+                    <QuickStartTile
+                      isActive={id === activeQuickStartID}
+                      quickStart={quickStart}
+                      status={getQuickStartStatus(allQuickStartStates!, id)}
+                    />
+                  </GalleryItem>
+                )
+              })}
+          </Gallery>
+        </QuickStartCatalogSection>
+        <QuickStartCatalogSection>
+          <TextContent>
+            <Text className="catalog" component="h2">Common instance for creating and tracking APIs</Text>
```

Comment:
I still don't understand the "Common instance" term... but maybe that's because I'm used to a different meaning of "instance" 
I would think of something like "Common scenarios" or "Common use cases".

But if approved by @dfennessy then it's OK I think :)